raiwidgets/raiwidgets/explanation_dashboard.py

```diff
@@ -25,7 +25,8 @@ class and for the regression case a method of predict()
         (# examples x # features),
         the same samples used to build the explanation.
         Overwrites any existing dataset on the explanation object.
-        Must have fewer than 10000 rows and fewer than 1000 columns.
+        Must have fewer than 100000 rows and fewer than 1000 columns.
+        Note dashboard may become slow or crash for more than 10000 rows.
```

Comment:
This seems scary!

Reply:
it's just making the docs reflect the code, which the user was confused about